.circleci/config.yml

```diff
@@ -103,7 +103,7 @@ commands:
                           ${ENABLE_ENTERPRISE:+-enable-enterprise} \
                           -enable-multi-cluster \
                           -debug-directory="$TEST_RESULTS/debug" \
-                          -consul-image=docker.mirror.hashicorp.services/hashicorp/consul-enterprise:1.12.0-beta1-ent \
+                          -consul-image=hashicorpdev/consul \
```

Comment:
This is the only build currently that has the fix in consul that was merged yesterday.  When that version is published, I'll remove this and merge this PR.